tests/fv3_conf/control_run.IN

```diff
@@ -13,10 +13,16 @@ fi
 echo "inputdir=$inputdir,NPX=$NPX"
 
 OPNREQ_TEST=${OPNREQ_TEST:-false}
+V2_SFC_FILE=${V2_SFC_FILE:-false}
 SUFFIX=${RT_SUFFIX}
 if [ $WARM_START = .false. ]; then
   mkdir INPUT RESTART
-  cp -r @[INPUTDATA_ROOT]/${inputdir}/INPUT_L127/* ./INPUT/.
+  if [ "$V2_SFC_FILE" = "true" ]; then
```

Comment:
The 'coldstart' data for the new tests currently resides in a personal directory. These files will need to be placed in the official directory. Then, I can update this file.

Reply:
Are these data replacing/overwriting older ones, or brand new?

Reply:
I see your comments in the PR, nevermind.